Sources/SQLite/Typed/Coding.swift

```diff
@@ -387,7 +387,11 @@ private class SQLiteDecoder: Decoder {
             } else if type == Date.self {
                 let date = try row.get(Expression<Date>(key.stringValue))
                 return date as! T
+            } else if type == UUID.self {
+                let uuid = try row.get(Expression<UUID>(key.stringValue))
+                return uuid as! T
             }
+
```

Comment:
As the number of branches get bigger, it might be better to use a switch statement here:
```swift
switch type {
case is Data.Type:
...
```

The same actually for encode:
```swift
func encode<T>(_ value: T, forKey key: Key) throws where T: Swift.Encodable {
    switch value {
    case let data as Data:
    ...
```